core/market/decentralized/src/matcher/store.rs

```diff
@@ -92,6 +111,18 @@ impl SubscriptionStore {
             .collect())
     }
 
+    pub async fn get_offers_batch(
+        &self,
+        ids: Vec<SubscriptionId>,
```

Comment:
this should be also more generic if you'll decide to follow my comment on `OffersDao`

Reply:
I think the opposite. The bottom layer (Dao) should have the most generic functions as we can make. Top layers like Store should have specialized functions that hide unused details.
It makes `Store` a little bit more useful

Reply:
👍 agreed